alerts/README.md

```diff
@@ -205,6 +214,14 @@ alertmanagerConfiguration:
   name: global-alertmanager-configuration
 ```
 
+## TLS Certificate Requirement
+
+In order for Prometheus installations in onboarded Greenhouse clusters to communicate with the Alertmanager component, the Ingress requires a TLS certificate for which Prometheus is configured as trusted. There are various ways in which you can generate/configure the required TLS certificate.
```

Comment:
Hard to wrap it in words. Tried to make it a little bit more explanatory, especially the first sentence. 

```suggestion
Greenhouse onboarded Prometheus installations need to communicate with the Alertmanager component to enable advanced processing of alerts. The Alertmanager Ingress requires a TLS certificate to be configured and trusted by Prometheus to ensure the communication. There are various ways in which you can generate/configure the required TLS certificate.
```